.travis.yml

```diff
@@ -11,32 +11,42 @@ services:
   
 before_install:
   # - setup_docker.sh
+  - mvn install:install-file -Dfile=datavault-common/src/main/resources/ftm-api-2.4.2.jar -DgroupId=oracle.cloudstorage.ftm -DartifactId=ftm-api -Dversion=1.0 -Dpackaging=jar
+  - mvn install:install-file -Dfile=datavault-common/src/main/resources/low-level-api-core-1.14.19.jar -DgroupId=oracle.cloudstorage.ftm -DartifactId=low-level-api-core -Dversion=1.0 -Dpackaging=jar
   - docker-compose rm -f
   - docker-compose pull
 
 script: 
   # docker needs jar file to start
+  - echo "Got here 1"
```

Comment:
I'm not sure why you've put "got here" messages, Travis display the command being run before running them:
```
$ docker-compose build
```
so I don't think that's helpful?